x11/Hypr/Portfile

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           github 1.0
+PortGroup           legacysupport 1.1
+
+# _strlen, _strdup
+legacysupport.newest_darwin_requires_legacy 10
+
+github.setup        hyprwm Hypr 855ef848fe0aded503b4100e734e58a3b872a91f
```

Comment:
as usual, this should be using a tagged version. If you want to provide a tag that should be a -devel port.

Reply:
It does not build from the tag, see the issue. It uses a function which does not exist in MacOS.

P. S. I have a reason to make it from a commit when I do so with my ports.

Reply:
@reneeotten Could we please merge this to reduce the queue? (There is nothing Python here.)

Reply:
Upstream agreed to make a release, so I will update to that: https://github.com/hyprwm/Hypr/issues/127#issuecomment-2024280099

Reply:
@reneeotten I have switched to released version now.